blocks/rich-text/style.scss

```diff
@@ -82,7 +82,7 @@
 
 .block-rich-text__inline-toolbar {
 	display: flex;
-	justify-content: center;
+	justify-content: attr( data-justify-content );
```

Comment:
@jasmussen I don't know if this is a good way to address the issue where the toolbar should be aligned the same as the content? Looking at the `text-align` property seems too hard to do.

Reply:
This is a bit hard to grok for me without a screenshot. I'm also not familiar with the `justify-content` property. Is this about the alignment of the inline toolbar in context of that text being non-centered?

Reply:
@jasmussen Sorry, I changed it in the meantime because this didn't work... I've added the styles inline in React now. Ideally the toolbar should be positioned the same as the text, but it doesn't seem like there's an easier way to do it. The block has to set the toolbar position as well.